src/autogluon/bench/runbenchmark.py

```diff
@@ -130,6 +126,27 @@ def run_benchmark(
     _dump_configs(benchmark_dir=benchmark.metrics_dir, configs=configs, file_name="configs.yaml")
 
     if configs.get("METRICS_BUCKET", None):
+        framework_name = module_kwargs["run_kwargs"]["framework"].split(":")[0]
+        upload_path = []
+        if module_name == "multimodal":
+            upload_path = [
+                framework_name,
+                module_kwargs["run_kwargs"]["dataset_name"],
+                module_kwargs["run_kwargs"]["constraint"],
+                configs.get("mode", None),
+            ]
+        else:
+            upload_path = [
+                framework_name,
+                module_kwargs["run_kwargs"]["benchmark"],
+                module_kwargs["run_kwargs"]["task"],
+                module_kwargs["run_kwargs"]["constraint"],
+                configs.get("mode", None),
+                module_kwargs["run_kwargs"]["fold"],
+            ]
```

Comment:
Maybe we should catch the situation where `module_name` falls outside of `multimodal`, `timeseries` and `tabular`

Reply:
yup makes sense, handled.
for now I have kept it like multimodal